apiary.apib

```diff
@@ -744,6 +744,7 @@ Add a new contact.
         + bic: `BICBANK` (string, optional)
         + remarks: `Met at expo` (string, optional)
         + tags: prospect, expo (array[string], optional)
+        + opted_in_for_mailings: false (boolean, optional)
```

Comment:
Q: Are there going to be more booleans like this? Should this be bundled under some kind of privacy settings? Just want to make sure we asked all those questions before setting ourselves up for a potential BC break in the future. @jenssegers ?

Reply:
It's actually already returned in `contacts.info` and `companies.info` like this, so I'd like to keep add/update consistent with the returned data if possible.

Reply:
@WouterSioen Unless we also change it in the add/update. I still feel like this is a sub-optimal solution. Wouldn't mind hearing @jdrieghe's possible alternative.

Reply:
In the one hand, I like the explicitness of the naming, in the other hand it feels wrong for some reason...

Maybe because a past tense feels weird for a property name? Properties are about state, not what happend before (that would be events)...

Would `mailing_opt_it` be more logical? In GDPR I read : `consent` from time to time... so `consent_to_mail`, ... maybe we could take this further and say : 

```
   ...
    "consent" : {
      "mail" : true,
      "phone" : false
    }
    ...
```

Reply:
Another train of thought, having something like contact preferences:

```
contact_preferences: [EMAIL, PHONE]
```

Or if you don't want to receive any communication:

```
contact_preferences: [NONE]
```

Reply:
Not sure if it's useful to introduce a BC break for this, since the property is already returned like this for a while in the contact and company `.info` endpoints.